Bio/motifs/__init__.py

```diff
@@ -25,15 +25,15 @@ def create(instances, alphabet=None):
 
 def parse(handle, format):
     """Parse an output file from a motif finding program.
-
```

Comment:
Please restore this blank line, expected under PEP257, which is causing this style check failure:

```
Bio/motifs/__init__.py:26:1: D205 1 blank line required between summary line and description
```